src/gps/GPS.cpp

```diff
@@ -252,6 +252,11 @@ bool GPS::setup()
 {
     int msglen = 0;
 
+    if (config.position.gps_mode == meshtastic_Config_PositionConfig_GpsMode_NOT_PRESENT) {
```

Comment:
I think this could be checked already in runOnce() just before setup() is called, so that disable() is executed.